verilog/analysis/verilog_project.cc

```diff
@@ -185,6 +186,24 @@ bool VerilogProject::RemoveRegisteredFile(
   return false;
 }
 
+std::string VerilogProject::GetRelativePathToSource(
+    const std::string& absolute_filepath) {
```

Comment:
Does this need to be `const std::string&` ? Ideally we use absl::string_view wherever possible as parameter, so that we can avoid awkward (and expensive) conversions to strings such as in line 200